### PR TITLE
Update event

### PR DESCRIPTION
**Pull request by azy64**
this pull request is about to add some updating events. The following requirements have been done :
- General requirements :heavy_check_mark: 
- HTML/CSS requirements :heavy_check_mark: 
- JavaScript requirements :heavy_check_mark: 
- Project requirements: Add a new JavaScript file and import it as a module, Add event listener to the checkbox, Update items object's value for completed key upon user actions, Store the updated array of items in local storage, so the user gets the correct list values after the page reloads :heavy_check_mark:

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -44,11 +46,21 @@ const component = () => {
   element.textContent = _.join(['Made', 'By SAIDI AZARIA'], ' ');
   return element;
 };
+/**
+ * this function load images and  give them
+ * some events
+ */
 const loadIconAndEvent = () => {
   const squares = document.querySelectorAll('.square');
```
Comment (EricMbouwe) [inline src/index.js]: - [ ] Nice work done here with the square images but for this project, you are required to use the `input` tag to create the [checkbox](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox). You will need to implement the status updates (completed: true/false) feature using the `change` event listener in one file and import it as a module in your main file.

Here is a picture of these requirements

Screenshot from 2021-09-30 10-51-43

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @azy64,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 import img1 from './images/refresh-hover.png';
 import enter from './images/enter.png';
-import square from './images/square.png';
-import squareHover from './images/square-hover.png';
+// import square from './images/square.png';
+// import squareHover from './images/square-hover.png';
 import menu from './images/menu.png';
 import menuHover from './images/menu-hover.png';
-import checked from './images/checked.png';
+// import checked from './images/checked.png';
```
Comment (Emmyn5600) [inline src/index.js:10]: #### Optional suggestion

 - [ ] Remove the commented line

File: src/update.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            target.classList.add('line-through');
+            data[num].completed = true;
+          } else {
+            target.classList.remove('line-through');
+            data[num].completed = false;
+          }
+          this.save(data);
+        });
+      });
+      // console.log('vraiment:', checkboxs);
```
Comment (Emmyn5600) [inline src/update.js:31]: #### Optional suggestion

 - [ ] Remove the commented line

File: src/index.js [hunk 4]
```diff
@@ -98,6 +91,12 @@ const editable = () => {
     element.addEventListener('blur', () => {
       const parent = element.parentNode;
       // console.log('jai perdu le focus');
+      element.classList.add('line-through');
+      const imgId = parseInt(parent.previousSibling.previousSibling.firstChild.nextSibling.id, 10);
+      // console.log('image:', imgId);
+      const NUM = data.findIndex((value) => value.index === imgId);
+      data[NUM].description = element.textContent;
```
Comment (Emmyn5600) [inline src/index.js:98]: #### Optional suggestion

 - [ ] Remove the commented line

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @azy64 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.

rr21
 #### Optional suggestions
 
 -N/A
   
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**